UnitsNet.Tests/CustomCode/ForceTests.cs

```diff
@@ -49,13 +49,34 @@ public void ForceDividedByAreaEqualsPressure()
         }
 
         [Test]
+        public void PressureByAreaEqualsForce()
+        {
+            Force force = Force.FromPressureByArea(Pressure.FromNewtonsPerSquareMeter(9), Area.FromSquareMeters(9));
+            Assert.AreEqual(force, Force.FromNewtons(81));
+
+            force = Force.FromPressureByArea(Pressure.FromNewtonsPerSquareMeter(9), Length2d.FromMeters(3, 3));
```

Comment:
It is not very important to me, but I would consider breaking this up into two tests. One with Area and one with Length2d. Just add a suffix to the test method name, such as `PressureByAreaEqualsForce_Length2d`. Same for the other test `MassByAccelerationEqualsForce`.